### PR TITLE
1224 editorial updates for section & article

### LINKED ISSUE #1224: HTML structure <article> and <section> needs to be redone
**Structure of document HTML 5.2 very strange and implicitly and must be rewrite**. It's about elements `<article>` and `<section>`. What we have now. - example of structure what html creators think is perfect:

```
<article>
	<header>
		<h2>Apples</h2>
		<p>Tasty, delicious fruit!</p>
	</header>
	<p>The apple is the pomaceous fruit of the apple tree.</p>
	<section aria-label="Red apples.">
		<h3>Red Delicious</h3>
		<p>These bright red apples are the most common found in many
		supermarkets.</p>
	</section>
	<section aria-label="Green apples.">
		<h3>Granny Smith</h3>
		<p>These juicy, green apples make a great filling for
		apple pies.</p>
	</section>
</article>
```

And same example as it should be (content(article) "Apples" inside big category(section) and sub-categories(section) "Red Delicious" and "Granny Smith" inside big category(section)):
```
<section>
	<article>
		<header>
			<h2>Apples</h2>
			<p>Tasty, delicious fruit!</p>
		</header>
		<p>The apple is the pomaceous fruit of the apple tree.</p>
	</article>
	<section aria-label="Red apples.">
		<h3>Red Delicious</h3>
		<p>These bright red apples are the most common found in many
		supermarkets.</p>
	</section>
	<section aria-label="Green apples.">
		<h3>Granny Smith</h3>
		<p>These juicy, green apples make a great filling for
		apple pies.</p>
	</section>
</section>
```

 
or this (big category(section) "Apples" and sub-categories(section) "Red Delicious" and "Granny Smith" inside big category(section)):

```
<section>
	<header>
		<h2>Apples</h2>
		<p>Tasty, delicious fruit!</p>
	</header>
	<p>The apple is the pomaceous fruit of the apple tree.</p>
	<section aria-label="Red apples.">
		<h3 aria-label="Red apples.">Red Delicious</h3>
		<p>These bright red apples are the most common found in many
		supermarkets.</p>
	</section>
	<section aria-label="Green apples.">
		<h3 aria-label="Green apples.">Granny Smith</h3>
		<p>These juicy, green apples make a great filling for
		apple pies.</p>
	</section>
</section>
```

Main problem that write HTML creators about `<article>` and `<section>` does not conform to the logic and common sense. Because always everywhere content placed inside category and not category inside content. I think **it must be strongly deprecated placed `<section>` inside `<artice>`**

### PR DESCRIPTION
**Pull request by scottaohara**
[Fixes #1224](https://github.com/w3c/html/issues/1224), in that these suggested updates to the prose for `article` and `section` are meant to further clarify the existing purposes for each element.

`article`s remain elements that contain a full composition, and `section`s remain elements that demarcate sub-sections of an `article`, document or application.

Example 7 was updated to include a brief example of when an `article` within a `section` element may be appropriate.

Some minor typos and source formatting are includes in these updates.

### CODE REVIEW

**Review by chaals — APPROVED** [review]
LGTM. Thank you

**Comment by chaals** [comment]
I didn't merge because I'd like a second review on this. Semantics is hard, because measuring it against e.g. browser implementation data is very hard and not very reliable...

File: sections/semantics-sections.include
Comment (prlbr) [inline sections/semantics-sections.include]: “implicitly implied” is tautological.

File: sections/semantics-sections.include
Comment (LJWatson) [inline sections/semantics-sections.include]: Add "a" into "the content is *a complete, or self-contained..."

**Comment by LJWatson** [comment]
LGTM. @scottaohara, feel free to merge once you've fixed the minor grammatical nit I noted :)

File: sections/semantics-sections.include
Comment (scottaohara) [reply sections/semantics-sections.include]: Thanks @LJWatson!  updated and will merge after checks finish. :)

**Pull request merged**
